ResearchKit/Common/ORKFormStep.h

```diff
@@ -129,6 +129,14 @@ ORK_CLASS_AVAILABLE
 @property (nonatomic, copy, readonly) NSString *identifier;
 
 /**
+ A Boolean value indicating whether the form item is optional.
+ 
+ The default value of this property is `NO`. When the value is `YES`, this form item doesn't need
```

Comment:
Thanks for the explanation.

Before this PR, the formItems are `All optional` or `All non-optional` controlled by form step optionality. (Not anymore in this PR)

So regardless the default value `YES` or `NO` we give to formItem optionality, we cannot make all existing behavior unchanged in the existing apps.

Like I said previously, to align with ORKStep's optionality for questionStep, make default value `YES` is better. Agree?